pueue/src/bin/pueue.rs

```diff
@@ -102,6 +103,15 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+// Clap **doesn't** support nushell
+// So I have written a completion script for nushell
+fn generate_nushell_completion(output_directory: &PathBuf) -> std::result::Result<PathBuf, std::io::Error>{
+    let path = output_directory.join("pueue.nu");
+    //It should be in pueue directory
+    fs::write(path.clone(), include_bytes!("pueue.nu"))?;
+    Ok(path)
+
+}
```

Comment:
To be honest, I'm not comfortable with having this in the generate command, since this creates the impression that this script will be auto-generated just like the others.

As soon as I'm going to change something on the CLI interface, this script will be outdated.


-> It would be great if you could revert all changes in the actual rust code. I think its best if that file iis located somewhere in the project to make sure that people understand that this won't update itself.